### DISCUSSION TITLE
Cant install python on it

### DISCUSSION THREAD

**Discussion by JustSomeTrash** [sherlock-project/sherlock#1130] [Q&A]
When I type in 
`python3 -m pip install -r requirements.txt`

it gives me an error
`/usr/bin/python3: No module named pip`

idk why, its supposed to work, yes i did cd into sherlock

**Comment by allordacia** [comment]
What OS are you running it on? It looks like you dont have pip installed: https://pip.pypa.io/en/stable/installation/#get-pip-py. Additionally, the context of your command doesnt make sense, what are you trying to do?

**Reply by franklinjosh11** [reply]
> What OS are you running it on? It looks like you dont have pip installed[:](https://borderpolar.com/best/best-unblocked-games/)  Additionally, the context of your command doesnt make sense, what are you trying to do?

this trick is working well try it i also tried this. when i was facing the issue

**Comment by chirag-droid** [comment]
On some systems pip doesn't come installed you have to install it

I think
`sudo apt install python3-pip` should fix it probably
